stackit/internal/services/argus/credential/resource.go

```diff
@@ -86,6 +86,7 @@ func (r *credentialResource) Configure(ctx context.Context, req resource.Configu
 
 func (r *credentialResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
 	resp.Schema = schema.Schema{
+		Description: "Argus credential resource schema. Must have a `region` specified in the provider configuration.",
```

Comment:
Maybe this could be a bit misleading as setting the custom endpoint could be enough

Reply:
Also do you think this should be maintained here? Is it not subject to change for each service?

Reply:
Adding a custom endpoint does override the need for the region. I'd leave it as is, because using a custom endpoint won't be done in normal use.

It is subject to change, but that requires changing the API, then the SDK, then the `Configure` routine of the resource